pytest.ini

```diff
@@ -1,5 +1,5 @@
 [pytest]
-addopts =
+xaddopts =
```

Comment:
should we revert this file?

Reply:
ARGHSD! I need to find a way to have it disabled locally without messing with this file. Reverted.